### DISCUSSION TITLE
(Airodump question/issue): Failed initializing wireless card(s): en0

### DISCUSSION THREAD

**Discussion by moon-dew** [aircrack-ng/aircrack-ng#2396] [Q&A]
I'm on a mac, and the current "BSD Device Name" which I believe is what should be inputted into airodump is en0, checked with a few tools and it's definitely active, but when i try running: `./airodump-ng -c 11 en0` it gives the error I put in the title of this question, and I'm really not sure where to go from here lol, I've spent the last hour or so trying to figure out how aircrack and airodump work at all so if this is a dumb question I apologize lol, but I thought this would be the best place to ask! Happy to offer any more information if anything would be relevant!

**Comment by Mister-X-** [comment]
AFAIK, it doesn't have any code to do packet capture on Mac. There is some code laying around from years ago, but it likely doesn't apply anymore, and considering Apple changes its API regularly, it likely does not compile or work anymore.

While we'd love to have it in the code, nobody in the team has a Mac at this time, so it won't happen anytime soon, and we'd have to rely on someone else to build it.

**Reply by moon-dew** [reply]
Oh i see, is there another tool that you know of that does packet capture for mac or am i kinda out of luck lol

**Reply by Mister-X-** [reply]
A couple:
- Builtin airport tool
- Wireshark

**Reply by moon-dew** [reply]
> A couple:
> 
> * Builtin airport tool
> * Wireshark

Thank you! Feel free to mark this as closed
